pkg/convert/tf.go

```diff
@@ -3251,12 +3278,55 @@ func translateModuleSourceCode(
 	return state.diagnostics
 }
 
+func getPackageBlock(name string, prov *configs.RequiredProvider) (*hclwrite.Block, hcl.Diagnostics) {
+	// if the name is one of our known providers just write it, if it is not it
+	// must be tf so write a paramaterized package.
+
+	// 	package <name> {
+	//   baseProviderName = <name>
+	//   baseProviderVersion = <version>
+	//   baseProviderDownloadUrl = <url>
+	//   parameterization {
+	//       name = <name>
+	//       version = <version>
+	//       value = <base64-encoded-value>
+	//   }
+	// }
```

Comment:
Maybe an opportunity to pull this out to a doc comment?

Reply:
indeed